.gitignore

```diff
@@ -1,3 +1,4 @@
 target
 Cargo.lock
 src/main.rs
+.idea
```

Comment:
Can this change be removed? Tracking the ignore settings for various tools would be an endless collection that I don't think each project should be tracking. If possible, I recommend putting this in a global ignore, or use local settings.

Reply:
Fair enough, I'll revert this change.